scottypy/beam.py

```diff
@@ -2,6 +2,8 @@
 import json
 from pact import Pact
 
+from scottypy.utils import raise_for_status
+
```

Comment:
Too many blank lines

Reply:
Actually, there are supposed to be 2 empty lines between imports and rest of code